### ISSUE TITLE
Redesign af header

### ISSUE THREAD

**Issue by andreasemil** [Pilen/fdfmaerker#15]

**Comment by andreasemil** [comment]
![skaermbillede 2018-11-03 kl 20 09 17](https://user-images.githubusercontent.com/43680974/47956444-eab28a00-dfa4-11e8-85fd-f69641f70153.png)

**Comment by Pilen** [comment]
Har du ændret på stjernernes størrelse på billedet?

**Comment by andreasemil** [comment]
Nej, det er headeren der er anderledes. Skjoldet i venstre side, align med indhold og mærkedatabasen.dk til højre for

**Comment by Pilen** [comment]
Hmm, det er fordi det er et unicode tegn, forskellige browsere falder tilbage på forskellige fonte. Kan se at på min telefon er den metalfarvet.
